.github/workflows/sphinx_build_on_release.yml

```diff
@@ -0,0 +1,27 @@
+name: Sphinx Build on Release
+
+on: release
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Install docs requirements
+        run: |
+          pip install -r docs/source/requirements.txt
+
+      # Why do we have to do this?
```

Comment:
why is this comment here?

Reply:
Shouldn't the additional package also be specified in docs/requirements.txt ?

Reply:
yes it should be in the requirements.txt file

Reply:
RTD already performs some installation by default:

```
 /home/docs/checkouts/readthedocs.org/user_builds/intelowl/envs/develop/bin/python -m pip install --upgrade --no-cache-dir mock==1.0.1 pillow==5.4.1 alabaster>=0.7,<0.8,!=0.7.5 commonmark==0.8.1 recommonmark==0.5.0 sphinx<2 sphinx-rtd-theme<0.5 readthedocs-sphinx-ext<2.2 
```

It does that before installing the requirements in the `docs` folder. So yeah, there works but in the github action is needed.

Reply:
@mlodic Is it alright to add it to docs/source/requirements.txt?